lzy/allocator-api/src/main/proto/ai/lzy/v1/allocator/volume.proto

```diff
@@ -36,6 +37,12 @@ message DiskVolumeType {
     READ_WRITE_MANY = 3;
     READ_ONLY_MANY = 4;
   }
+
+  enum StorageClass {
+    DEFAULT = 0;
```

Comment:
plz, don't use default values in proto (there is no difference between default value and no value)

let's rename it to `STORAGE_CLASS_UNSPECIFIED`